src/rules/whitespaceRule.ts

```diff
@@ -27,6 +27,7 @@ const OPTION_DECL = "check-decl";
 const OPTION_OPERATOR = "check-operator";
 const OPTION_MODULE = "check-module";
 const OPTION_SEPARATOR = "check-separator";
+const OPTION_SPREAD = "check-spread";
```

Comment:
I'd prefer `check-rest-spread` which checks object and array spread as well as object and array rest and rest parameters